src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Providers/Providers/IaasVmPsBackupProvider.cs

```diff
@@ -77,5 +78,113 @@ public void DeletePolicy()
         {
             throw new NotImplementedException();
         }
+
+
+        public AzureRmRecoveryServicesSchedulePolicyBase GetDefaultSchedulePolicyObject()
+        {
+            AzureRmRecoveryServicesSimpleSchedulePolicy defaultSchedule = new AzureRmRecoveryServicesSimpleSchedulePolicy();
+            //Default is daily scedule at 10:30 AM local time
+            defaultSchedule.ScheduleRunFrequency = ScheduleRunType.Daily;
+
+            DateTime scheduleTime = GenerateRandomTime();
+            defaultSchedule.ScheduleRunTimes = new List<DateTime>();
+            defaultSchedule.ScheduleRunTimes.Add(scheduleTime);
+
+            defaultSchedule.ScheduleRunDays = new List<DayOfWeek>();
+            defaultSchedule.ScheduleRunDays.Add(DayOfWeek.Sunday);
```

Comment:
If it is Daily schedule, we should not add WeekDay Sunday here. Remove this line.